Makefile

```diff
@@ -111,15 +111,18 @@ else
 BUILD_CONFIG  := Release
 endif
 
+ifeq ($(MAC), 1)
+# Only use ZSTD for the deb package if we are targeting mac
```

Comment:
Again, don’t change this: Pro users.

Reply:
Benefitting Pro users by resulting in a bigger deb?

Reply:
Marginally bigger with a non-marginally smaller decompression time.